pkg/aws/ec2.go

```diff
@@ -239,6 +239,9 @@ func (r AwsDynamicConfig) checkInstanceConnectivity(ctx context.Context, instanc
 }
 
 func pingSSHIp(ctx context.Context, ipAddress string) (string, error) {
+	// THIS CODE WILL BE REMOVED SOON
+	log := logr.FromContextOrDiscard(ctx)
+	log.Info("Just adding some code changes to test SeaLights")
```

Comment:
let's make sure this does not end up in prod, this will pollute the logs

Reply:
Of course, on it tomorrow morning